force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -241,8 +319,12 @@ public with sharing class FlowControlLoopsCollections {
         Set<String> stringSet = new Set<String>();
 
         //add strings to the set
+        stringSet.add( 'a');
+        stringSet.add( 'b');
+        stringSet.add( 'a');
+        stringSet.add( 'c');
 
-        return null; // Replace null with the variable you used to store the result
+        return stringSet; // Replace null with the variable you used to store the result
```

Comment:
You need to take the values passed in with the argument inputList and `dedupe` them with a set. You are hard coding the values in. Feel free to message me on Sack if you have a question about this.